index.d.ts

```diff
@@ -1,6 +1,52 @@
 import { EventEmitter } from "events";
 
-interface BasePromptOptions {
```

Comment:
This diff looks a bit odd b/c for git the smallest way to express the change is to actually move the start of the namespace and the classes above it to the top of the file, rather than moving the interfaces into the namespace.

So the visualisation of this diff is inverse to it's result :)